spec/rubocop/cop/style/braces_around_hash_parameters_spec.rb

```diff
@@ -181,55 +181,55 @@
       )
       RUBY
       corrected = autocorrect_source(src)
-      expect(corrected).to eq(<<-RUBY.strip_indent)
+      expect(corrected).to eq(<<-RUBY)
       foo(
-        qux: "bar",
+          qux: "bar",
           baz: "bar",
           thud: "bar"
       )
       RUBY
     end
 
     it 'corrects brace removal with lower extra line' do
-      src = <<-RUBY.strip_indent
+      src = <<-RUBY
       foo({
         baz: 7
         }
       )
       RUBY
       corrected = autocorrect_source(src)
-      expect(corrected).to eq(<<-RUBY.strip_indent)
+      expect(corrected).to eq(<<-RUBY)
       foo(
         baz: 7
       )
       RUBY
     end
 
     it 'corrects brace removal with top extra line' do
-      src = <<-RUBY.strip_indent
+      src = <<-RUBY
       foo(
         {
           baz: 5
       })
       RUBY
       corrected = autocorrect_source(src)
-      expect(corrected).to eq(<<-RUBY.strip_indent)
+      expect(corrected).to eq(<<-RUBY)
       foo(
-        baz: 5
+          baz: 5
```

Comment:
But this seems like a regression in functionality.

---

Do you think we can figure out a way to get consistent and correct alignment? There is a mixin called `Alignment` that may be helpful.